permissions/plugin-dingding-notifications.yml

```diff
@@ -2,6 +2,6 @@
 name: "dingding-notifications"
 github: "jenkinsci/dingding-notifications-plugin"
 paths:
-- "com/ztbsuper/dingding-notifications"
+  - "org/jenkins-ci/plugins/dingding-notifications"
```

Comment:
If you are taking over the existing plugin, is the path really changing? Are you going to modify the groupId from the pom.xml?

Reply:
> If you are taking over the existing plugin, is the path really changing? Are you going to modify the groupId from the pom.xml?

I will do it after take over the plugin.

Reply:
Better to use `io.jenkins.plugins` for new plugins